### DISCUSSION TITLE
Need help with my renovate config in my minimal reproduction repo

### DISCUSSION THREAD

**Discussion by OuesFa** [renovatebot/renovate#15687] [Request Help]
Hi,
As described here https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md
I've created a mini reproduction repo in which I describe what I would like Renovate to do in [the Readme file](https://github.com/OuesFa/test-renovate-config/blob/renovate/configure/README.md) 
Here is the [config file](https://github.com/OuesFa/test-renovate-config/blob/renovate/configure/renovate.json5) 
https://github.com/OuesFa/test-renovate-config/pull/1

For now it does nothing. 
Thanks!

**Comment by OuesFa** [comment]
Hello @rarkins 👋 
Sorry to ping you directly, I wanted to know if my request for help lacked some details that made it difficult to answer? 
Thanks for your help!

**Reply by rarkins** [reply]
Pinging anyone after one day is rude, please don't do that again.
